### ISSUE TITLE
Serve Static Service does not fail setup when port is busy

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-toolkit-boneyard#30]
# Bug Report

## Description

The server-static-service is not failing when the specified port is in use. Instead it continues to pack assets and run tests, resulting in all test failures.

## Steps to Reproduce

1. start server or docker service that uses port 8080
2. run wdio tests with default ports & watch it fail to use port 8080 but continue to compile and make the service start as successful
    - what happens is the wdio runner will run each test and end up with an error screenshot for each test

## Additional Context / Screenshots

```sh
> terra-alert@4.26.0 wdio-default /Users/er047227/terra/terra-core/packages/terra-alert
> cd ../.. && npx wdio ./packages/$npm_package_name/wdio.conf.js


[Terra-Toolkit:terra-service] Running tests against Selenium 3.14
[terra-aggregate-translations] Aggregating translations for en,en-AU,en-CA,en-US,en-GB,es,es-US,es-ES,de,fr,fr-FR,nl,nl-BE,pt,pt-BR,sv,sv-SE locales.
[Terra-Toolkit:theme-aggregator] Aggregating clinical-lowlight-theme files...
[Terra-Toolkit:theme-aggregator] Successfully generated scoped-clinical-lowlight-theme.scss.
[Terra-Toolkit:theme-aggregator] Aggregating orion-fusion-theme files...
[Terra-Toolkit:theme-aggregator] Successfully generated scoped-orion-fusion-theme.scss.
[Terra-Toolkit:theme-aggregator] Successfully generated aggregated-themes.js.
listen EADDRINUSE: address already in use 0.0.0.0:8080
[Terra-Toolkit:selenium-docker] Deploying docker selenium stack
[Terra-Toolkit:selenium-docker] Ensuring selenium status is ready
asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  temp-themes.css (352 KiB),entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  theme (353 KiB)
      temp-themes.css
      theme.js
,webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
[Terra-Toolkit:serve-static-service] Hot reloading has been disabled for tests.

```
## Expected Behavior

The server-static-service should fail if the specified port returns the error that the port is already in use.

## Environment

* Component Name and Version: `v6`
* Node/npm Version: `v10.18.0 / 6.13.4`
* Webpack Version: `4.43.0`

**Comment by avinashg1994** [comment]
After some investigation, here is what i found:
1. The "serveStaticService" fails when the port (8080) is in use due to the default port being passed as 8080 [here](https://github.com/cerner/terra-toolkit-boneyard/blob/master/src/wdio/services/ServeStaticService.js#L103). If removed, the port number increments like in "serve"
2. I also tried to read for "EADDRINUSE" error [here](https://github.com/cerner/terra-toolkit-boneyard/blob/master/src/wdio/services/ServeStaticService.js#L84), but it does not seem to catch that error here. The logic for detecting the server in use is in "webpack devserver".
3. Also trying a [third party approach](https://stackoverflow.com/questions/19129570/how-can-i-check-if-port-is-busy-in-nodejs) to detect if 8080 is use, but this to does not seem to work.

**Comment by emilyrohrbough** [comment]
I did some more digging into wepback-dev-server. It looks like they are [swallowing the error thrown when the port is busy instead of re-throwing the error](https://github.com/webpack/webpack-dev-server/blob/master/lib/Server.js#L692).

The error is swallowed both when using webpack-dev-server via the CLI and when implementing the server directly. 

I have logged an issue to their repo to see if we can get some traction: https://github.com/webpack/webpack-dev-server/issues/2624
